### DISCUSSION TITLE
Cannot find module or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by mooyg** [pnpm/pnpm#5634] [Q&A]
Hey my VS Code keeps bugging me with this although everything works fine and I get no errors while running my project. But In VSCode it keeps complaining

**Comment by lvqq** [comment]
Try reload VSCode window

**Reply by mooyg** [reply]
Already did didn't work

**Reply by lvqq** [reply]
Could u provide a code to reproduce it?

**Reply by mooyg** [reply]
I mean what code? their is no code it just doesn't recongize my module everything else works fine
